CONTRIBUTING.md

```diff
@@ -6,10 +6,10 @@ For big feature it's advised to raise an issue to discuss it first.
 ## TLDR: 
 
 * Install [the .NET Core SDK](https://dot.net/) 
-* run `./build.sh` on macOS/Linux or `powershell ./build.ps1` on Windows.
```

Comment:
I'm not sure if that's still recommended by you guys, but the `./build.ps1` did nothing good for me, it just blew away my nuget cache and then I couldn't build anything until closing the IDE and running `dotnet build`

Reply:
The build script will compile in release at least so those warnings as error for missing code docs will alert you.

Why did it blow up your nuget cache? The build script is just doing dotnet build

Reply:
Because of this code inside of build.sh
`dotnet nuget locals all --clear`

Reply:
Yep that'd be why. I'm not sure what exactly the reason is, but I've actually had a bunch of weird issues building locally. I've found that I can _usually_ get it to work if I manually run `dotnet restore` before building, but not always. The most common issue I've had is that it seems to be including the `Nullable` attributes for some reason 🤷🏻‍♂️

Reply:
That should definitely be removed

Reply:
https://github.com/getsentry/sentry-dotnet/commit/5626477b2b558c56bbd078a9b4163e2c6f731a05